doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -61,6 +61,78 @@ Alternatively, you can enable CORS using the AWS S3 web interface, using json-en
 
 Since the direct upload mechanism creates the final file rather than an intermediate temporary file, user actions, such as neither saving or canceling an upload session before closing the browser page, can leave an abandoned file in the store. The direct upload mechanism attempts to use S3 Tags to aid in identifying/removing such files. Upon upload, files are given a "dv-state":"temp" tag which is removed when the dataset changes are saved and the new file(s) are added in the Dataverse installation. Note that not all S3 implementations support Tags: Minio does not. WIth such stores, direct upload works, but Tags are not used.
 
+Trusted Remote Storage with the ``remote`` Store Type
```

Comment:
Would it make more sense for this portion of the big-data guide, that is not specific to Globus to be part of the base "remote stores" PR (7325)?

Reply:
I copied the text across to #7325.

Reply:
Not crucial, but how come I'm not seeing the change you mentioned above when I try to compare the 2 branches, like this: 
https://github.com/GlobalDataverseCommunityConsortium/dataverse/compare/IQSS/7324_TRSA-HTTP-store...GDCC/DC-1
what am I doing wrong?

Reply:
I'm not sure. FWIW: Because I added the remote store and Globus doc in one commit to this PR, I added the subset about remote store to the other PR as a separate/new commit. So perhaps I have to merge the remote store branch back into this for the change to be accurate in github?